packages/signing/android/src/main/java/org/iota/mobile/Interface.java

```diff
@@ -0,0 +1,13 @@
+package org.iota.mobile;
```

Comment:
Should this be `org.iota.Signing`?

Reply:
It should and we'll have to separate pow, hashing and signing over the next days.

Reply:
renamed in https://github.com/iotaledger/iota.js/pull/355/commits/a09e7908381b1ed8dab7d0b7f7c1327f23455f89